tests/src/provider/providerservice.test.ts

```diff
@@ -36,9 +35,13 @@ describe("test provider service", () => {
     });
 
     test("test get backups", async () => {
-        let backups: BackupDetail[] = await providerService.getBackups();
-        expect(backups).not.toBeNull();
-        expect(backups).not.toEqual([]);
+        try {
+            let backups: BackupDetail[] = await providerService.getBackups();
+            expect(backups).not.toBeNull();
+            expect(backups).not.toEqual([]);
+        } catch (e) {
+            expect(e).toBeInstanceOf(NotFoundException);
+        }
```

Comment:
Invalid test because we expect two different valid behaviors. Is the valid result an HTTP 404 error or an empty structure?

Reply:
If we split this test case with two. It is a little complex. We need to create two backup services for line #41 and remove all backup services for line #43. So making a test case like this is a better choice.